crates/phactory/src/contracts/support.rs

```diff
@@ -190,7 +190,7 @@ impl FatContract {
         phala_mq::select! {
             next_cmd = self.cmd_rcv_mq => match next_cmd {
                 Ok((_, cmd, origin)) => {
-                    info!(target: "contract", "Contract {:?} handling command", self.id());
+                    info!("Contract {:?} handling command", self.id());
```

Comment:
What will happen if the target is not specified?

Reply:
It would use the module path as the target. `phactory::contracts` for example.

Reply:
Sounds good